docs/download_files.md

```diff
@@ -139,6 +139,46 @@ For more on the specific contents of multiplexed library `SingleCellExperiment`
 
 The [metadata file](#metadata) for multiplexed libraries (`single_cell_metadata.tsv`) will have the same format as for individual samples, but each row will represent a particular sample/library pair, meaning that there may be multiple rows for each `scpca_library_id`, one for each `scpca_sample_id` within that library.
 
+
+## Merged object downloads
+
+When downloading a full ScPCA project, you can choose to download each library as an individual file, or you can download {ref}`a single file containing all libraries merged into a single object<faq:INCOMING MERGED OBJECT SECTION>`. TODO!
```

Comment:
```suggestion
When downloading a full ScPCA project, you can choose to download data from all samples as individual files, or you can download {ref}`a single file containing all libraries from all samples merged into a single object<faq:INCOMING MERGED OBJECT SECTION>`. TODO!
```
Copying some of my thoughts from my review in #244, we should use samples in the language.